deploy/operator/setup_assisted_operator.sh

```diff
@@ -278,13 +278,28 @@ data:
     $(registry_config "$(get_image_without_tag ${ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE})" "${LOCAL_REGISTRY}/$(get_image_repository_only ${ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE})")
     $(registry_config "$(get_image_without_tag ${cli_image})" "${LOCAL_REGISTRY}/$(get_image_repository_only ${ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE})")
     $(registry_config "$(get_image_without_tag ${ironic_agent_image})" "${LOCAL_REGISTRY}/$(get_image_repository_only ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE})")
-    $(for row in $(kubectl get imagecontentsourcepolicy -o json |
-        jq -rc ".items[] | select(.metadata.name | test(\"${assisted_index_image}\")).spec.repositoryDigestMirrors[] | [.mirrors[0], .source]"); do
-      row=$(echo ${row} | tr -d '[]"');
-      source=$(echo ${row} | cut -d',' -f2);
-      mirror=$(echo ${row} | cut -d',' -f1);
-      registry_config ${source} ${mirror};
-    done)
+    $(
+      if kubectl get crd imagedigestmirrorsets.config.openshift.io &>/dev/null; then
```

Comment:
Is this set by dev-scripts or from one of our scripts? If it's dev-scripts, did you already see if they changed to use IDMS for their mirrors? I can take a look if you don't know!

Reply:
I am not sure to understand your question @CrystalChun. Are you asking if there is something that runs after this script that uses the generated IDMS or IDSP (other than the cluster itself)? I didn't find such a thing in this repository, and I didn't check dev-scripts. If you give me a hint about where to start looking I can dig.

Reply:
Ah I read this wrong, sorry! I thought it would only look for either the idms or icsp, but it looks for both. Apologies!